### PR TITLE
Added Package Settings

### PR DESCRIPTION
**Pull request by angelocala94**
- Added “polling” enable/disable setting
- Added “polling_interval” setting

Allows you to disable or edit the polling interval on the backups Nova page. 
Using DigitalOcean Spaces polling causes `503 Slow Down` response from DO APIs.

#9 solved.

### CODE REVIEW

File: src/BackupToolServiceProvider.php [hunk 1]
```diff
@@ -37,6 +49,18 @@ protected function routes()
             );
     }
 
+    protected function provideConfigToScript()
+    {
+        Nova::serving(function (ServingNova $event) {
+            Nova::provideToScript([
+                'novabackuptool' => [
```
Comment (freekmurze) [inline src/BackupToolServiceProvider.php]: rename this to `nova_backup_tool`

File: config/novabackuptool.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (freekmurze) [inline config/novabackuptool.php]: Rename the file to `nova-backup-tool.php`

File: config/novabackuptool.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
```
Comment (freekmurze) [inline config/novabackuptool.php]: Let's lose this header and `|` prefixes. Just a simple comment is ok.

**Comment by angelocala94** [comment]
All done @freekmurze

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**